src/main/java/com/schedule/workout/workoutScheduler/controller/jwtToken/JwtTokenController.java

```diff
@@ -0,0 +1,47 @@
+package com.schedule.workout.workoutScheduler.controller.jwtToken;
+
+import com.schedule.workout.workoutScheduler.exceptions.UnauthorizedException;
+import com.schedule.workout.workoutScheduler.services.JwtTokenService;
+import io.jsonwebtoken.ExpiredJwtException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.validation.Valid;
+
+
+@Controller
+public class JwtTokenController {
+    @Autowired
+    JwtTokenService tokenService;
+
+    @RequestMapping(method = RequestMethod.POST, value = "/token")
+    public ResponseEntity<?> userLogin(@Valid @RequestBody JWTRequest jwtRequest) {
```

Comment:
This should be called "createToken" as its doing just that. Jwt tokens are not necessarily tied to users.